Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.Methods.cs

```diff
@@ -174,9 +177,25 @@ private void RenderMarkdown()
             MarkdownRendered?.Invoke(this, markdownRenderedArgs);
         }
 
+        private void HookListeners()
+        {
+            // Re-hook all hyper link events we currently have
+            foreach (object link in _listeningHyperlinks)
+            {
+                if (link is Hyperlink hyperlink)
+                {
+                    hyperlink.Click += Hyperlink_Click;
```

Comment:
Do you mind if we are extra careful so we don't hook it up twice? Example https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.cs#L113-L117
I would also do it on the RegisterNewHyperLink methods.

Reply:
Done, please take a look at latest iteration.